v1/acl.go

```diff
@@ -0,0 +1,177 @@
+package v1
+
+import (
+	"context"
+	"errors"
+
+	client "github.com/hashicorp/nomad-openapi/clients/go/v1"
+)
+
+type ACL struct {
+	client *Client
+}
+
+func (c *Client) ACL() *ACL {
+	return &ACL{client: c}
+}
+
+func (a *ACL) ACLApi() *client.ACLApiService {
+	return a.client.apiClient.ACLApi
+}
+
+// returns arraySchema
+func (a *ACL) GetACLPolicies(ctx context.Context) (*[]client.ACLPolicyListStub, *QueryMeta, error) {
```

Comment:
Reminder that we should look at naming conventions in V1 vs the generated clients when we meet next week.